google-cloud-pubsub-v1/AUTHENTICATION.md

```diff
@@ -27,7 +27,7 @@ export PUBSUB_CREDENTIALS=path/to/keyfile.json
 ```ruby
 require "google/cloud/pubsub/v1"
 
-client = ::Google::Cloud::PubSub::V1::Publisher::Client.new
+client = ::Google::Cloud::PubSub::V1::SchemaService::Client.new
```

Comment:
@dazuma Is there a way to specify `Publisher` as the desired service for examples in this gem? (`SchemaService` is of marginal importance compared to `Publisher`.)

Reply:
This will be addressed by googleapis/gapic-generator-ruby#526